hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/SparkRDDTableServiceClient.java

```diff
@@ -291,6 +287,13 @@ private void completeClustering(HoodieReplaceCommitMetadata metadata,
     LOG.info("Clustering successfully on commit " + clusteringCommitTime);
   }
 
+  private void handleWriteErrors(List<HoodieWriteStat> writeStats, TableServiceType tableServiceType) {
+    if (writeStats.stream().mapToLong(HoodieWriteStat::getTotalWriteErrors).sum() > 0) {
+      throw new HoodieClusteringException(tableServiceType + " failed to write to files:"
+          + writeStats.stream().filter(s -> s.getTotalWriteErrors() > 0L).map(HoodieWriteStat::getFileId).collect(Collectors.joining(",")));
```

Comment:
wondering whether we need a config option to control the behavior? If all the exceptions are resolvable, it's okay we throw exceptions directly.

Reply:
I prefer to throw exception in this case to avoid unexpected behavior later.